hera_pspec/pstokes.py

```diff
@@ -86,8 +87,10 @@ def _combine_pol(uvd1, uvd2, pol1, pol2, pstokes='pI', x_orientation=None):
         Polarization of the second UVData object to use in constructing
         pStokes visibility.
 
-    pstokes: Pseudo-stokes polarization to form, type: str
-        Pseudo stokes polarization to form, can be 'pI' or 'pQ' or 'pU' or 'pV'.
+    pstokes: Pseudo-stokes polarization to form, type: str or int
```

Comment:
Even though you now also take an integer, I think it's good to list the possible strings.